### ISSUE TITLE
Add allow_wildcard_certificates argument to vault_pki_secret_backend_role

### ISSUE THREAD

**Issue by dmaes** [hashicorp/terraform-provider-vault#1419]
### Terraform Version
```
Terraform v1.1.8
on linux_amd64
+ provider registry.terraform.io/hashicorp/vault v3.4.1
```

### Affected Resource(s)
- vault_pki_secret_backend_role

### Expected Behavior
Add `allow_wildcard_certificates` argument that corresponds to https://www.vaultproject.io/api-docs/secret/pki#allow_wildcard_certificates

**Comment by tmatilai** [comment]
PR to add the attribute: #1697

**Comment by tmatilai** [comment]
This has been shipped in v3.17.0 and can be closed

**Comment by fairclothjm** [comment]
closed by https://github.com/hashicorp/terraform-provider-vault/pull/1910
